support-data-rest/src/test/java/org/uniprot/api/support/data/taxonomy/controller/TaxonomySearchControllerIT.java

```diff
@@ -30,20 +44,6 @@
 import org.uniprot.store.search.SolrCollection;
 import org.uniprot.store.search.document.taxonomy.TaxonomyDocument;
 
```

Comment:
Why the formatting of this file changed? COuld you please run our default formatter?

Reply:
This is the normal format change